chat/message-feedback/package.json

```diff
@@ -5,7 +5,6 @@
   "main": "./dist/umd/index.js",
   "module": "./dist/esm/index.js",
   "types": "./dist/types/index.d.ts",
```

Comment:
This is a bandaid. We'll need to come up with a more rigorous solution if we want to export and share stories between components.